pipettes/core/can_task_ht.cpp

```diff
@@ -0,0 +1,125 @@
+#include "can/core/freertos_can_dispatch.hpp"
```

Comment:
Is it clear to everyone what `ht` and `lt` are?

Reply:
Probably not. I'll add a comment in both files.